notifier.js

```diff
@@ -14,7 +14,7 @@ exports.sendEmail = function (email, subjectLine, slotDetails, callback) {
         from: String('Vaccine Checker ' + process.env.EMAIL),
         to: email,
         subject: subjectLine,
-        text: 'Vaccine available. Details: \n\n' + slotDetails
+        text: slotDetails
     };
     nodemailerTransporter.sendMail(options, (error, info) => {
```

Comment:
I hope this takes care of a misconfiguration (or anything) - and throws up if the mail could not be sent.